combine_libraries.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import argparse
+from pathlib import Path
+import shutil
+import sys
+
+import numpy as np
+import openmc.data
+
+
+def library_in_list(lib, lib_list):
+    """Returns whether the given nuclide library is present in the given list of libraries.
+
+    Does this by checking whether libraries have the same type and material list.
+    """
+    for lib_in_list in lib_list:
+        if isinstance(lib_in_list, type(lib)):
+            if lib['type'] == lib_in_list['type'] \
+               and np.array_equal(lib['materials'], lib_in_list['materials']):
+                return True
+    return False
+
+
+description = """
+Script to combine nuclide files from multiple OpenMC HDF5 libraries into a single library.
+
+"""
```

Comment:
```suggestion
"""


```
perhaps add some white space (pep8)